url-updater/src/main/java/com/devonfw/tools/ide/url/tool/quarkus/QuarkusUrlUpdater.java

```diff
@@ -46,4 +46,21 @@ protected void addVersion(UrlVersion urlVersion) {
     }
   }
 
+  @Override
+  public String getCpeVendor() {
+
+    return "quarkusio";
```

Comment:
See https://nvd.nist.gov/vuln/detail/CVE-2023-6267#vulnConfigurationsArea
```suggestion
    return "quarkus";
```